### DISCUSSION TITLE
@next feature release discussion

### DISCUSSION THREAD

**Discussion by BitHighlander** [shapeshift/hdwallet#483] [General]
Posting this thread as a placeholder for the next major branch features and architecture discussion.
(out of scope of this thread is asset additions)

ref: https://github.com/shapeshift/hdwallet/pull/354 PR

Things on my list:

1.) support for thorchain keystore import/export/migrate
2.) proposals for removing the mixin madness flying V's [ref:](https://github.com/shapeshift/hdwallet/blob/architecture-refactor/packages/hdwallet-native/src/native.ts#L128)
3.) Any features we can add to make xchain-js integration smoother/cleaner
4.) first class kk-bridge support with multi connecting multiplexing (bypassing claim device limitations)

**Comment by mrnerdhair** [comment]
For reference: Much of #354 has already been merged; the rest is now updated and pulled out as single-purpose PRs:
- #419 
- #420 
- #421
- #427
- #431 
- #432
- #434
- #435
- #436
